examples_test.go

```diff
@@ -0,0 +1,63 @@
+package flyway_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path/filepath"
+	"time"
+
+	"github.com/CyberOwlTeam/flyway"
+
+	"github.com/testcontainers/testcontainers-go"
+	tcnetwork "github.com/testcontainers/testcontainers-go/network"
+)
+
+func ExampleRunContainer() {
+	// runFlywayContainer {
+	ctx := context.Background()
+	nw, err := tcnetwork.New(ctx)
+	if err != nil {
+		log.Fatalf("failed to start network: %s", err) // nolint:gocritic
+	}
+	postgresContainer, err := createTestPostgresContainer(ctx, nw)
+	if err != nil {
+		log.Fatalf("failed to start postgres container: %s", err) // nolint:gocritic
+	}
+
+	flywayContainer, err := flyway.RunContainer(ctx,
+		testcontainers.WithImage(flyway.BuildFlywayImageVersion()),
+		tcnetwork.WithNetwork([]string{"flyway"}, nw),
+		flyway.WithDatabaseUrl(postgresContainer.getNetworkUrl()),
+		flyway.WithUser(defaultPostgresDbUsername),
+		flyway.WithPassword(defaultPostgresDbPassword),
+		flyway.WithConnectRetries(3),
+		flyway.WithTable("my_schema_history"),
+		flyway.WithGroup("my_group"),
+		flyway.WithTimeout(1*time.Minute),
+		flyway.WithMigrations(filepath.Join("testdata", flyway.DefaultMigrationsPath)),
+	)
+	if err != nil {
+		log.Fatalf("failed to start container: %s", err) // nolint:gocritic
+	}
+
+	// Clean up the container
+	defer func() {
+		if err := flywayContainer.Terminate(ctx); err != nil {
+			log.Fatalf("failed to terminate container: %s", err) // nolint:gocritic
+		}
+	}()
+	//}
```

Comment:
to remove?

Reply:
I thought that ... it came out of the generated template, if you look. there is a comment above with something too, not sure what the significance is tho (I just left it) 

```
// runFlywayContainer {
....
// }
```